deploy/examples/import-external-cluster.sh

```diff
@@ -65,11 +65,17 @@ function checkEnvVars() {
     echo "Providing both ROOK_EXTERNAL_ADMIN_SECRET and ROOK_EXTERNAL_USER_SECRET is not supported, choose one only."
     exit 1
   fi
+  if [ -n "$KUBECONTEXT" ]; then
```

Comment:
Looks good,

Can you add some pointers of the changes in the documentation,
 https://github.com/rook/rook/blob/master/Documentation/CRDs/Cluster/external-cluster.md#commands-on-the-k8s-consumer-cluster
 
 When we would need this, and we need to set a extra env variable for this